langchain_tools_demo/Dockerfile

```diff
@@ -14,7 +14,7 @@
 
 # Use the official lightweight Python image.
 # https://hub.docker.com/_/python
-FROM python:3.11-slim
+FROM python:3.10-slim
```

Comment:
We want this to be as new as possible to reduce the risk of security and vulnerability issues.